Change CLI docs (#306)
cmu-delphi/delphi-epidata: docs/api/covidcast-signals/chng.md

| --- | --- |
| `smoothed_outpatient_covid` | Estimated percentage of outpatient doctor visits with confirmed COVID-19, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time using a Gaussian linear smoother |
| `smoothed_adj_outpatient_covid` | Same, but with systematic day-of-week effects removed; see [details below](#day-of-week-adjustment) |

> | `smoothed_outpatient_cli` | Estimated percentage of outpatient doctor visits primarily about COVID-related symptoms, based on Change Healthcare claims data that has been de-identified in accordance with HIPAA privacy regulations, smoothed in time using a Gaussian linear smoother |

Reviewer: ```suggestion
| `smoothed_outpatient_cli` | Estimated percentage of outpatient doctor visits primarily about COVID-related symptoms, based on Change Healthcare claims data that has been aggregated in accordance with HIPAA privacy regulations, smoothed in time using a Gaussian linear smoother |
```

I feel this would be better, but if CH has specific language choices then ignore it.

Reviewer 2: alas it's their language 😞

Reviewer: No worries, :) looks good to go then

	correlated with that area's historical influenza activity, but are
	diagnostic codes not specific to influenza and can appear in COVID-19 cases.
- Mixed: Daily count of all unique outpatient visits with primary ICD-10 code of
	any of: {Z20.828, J12.9}. The occurance of these codes in an area is
	correlated to a blend of that area's COVID-19 confirmed case counts and
	influenza behavior, and are not diagnostic codes specific to either disease.
- Flu: Daily count of all unique outpatient visits with primary ICD-10 code of
	any of: {J09\*, J10\*, J11\*}. The asterisk `*` indicates inclusion of all
	subcodes. This set of codes are assigned to influenza viruses.

For the COVID signal, we consider only the *Denominator* and *Covid* counts.

For the CLI signal, if a patient has multiple visits on the same date (and hence
multiple primary ICD-10 codes), then we will only count one of and in descending
order: *Flu*, *COVID-like*, *Flu-like*, *Mixed*. This ordering tries to account for
the most definitive confirmation, e.g. the codes assigned to *Flu* are only used
for confirmed influenza cases, which are unrelated to the COVID-19 coronavirus.

## Estimation

> ### COVID Illness

Reviewer 2: ```suggestion
### COVID Illness

The following estimation method is used for the `*_outpatient_covid` signals.
```

\hat p_{it} = 100 \cdot  \frac{Y_{it}}{N_{it}}
$$

> ### COVID-Like Illness

Reviewer 2: ```suggestion
### COVID-Like Illness

The following estimation method is used for the `*_outpatient_cli` signals.
```
